src/SleepForDays/SleepForDays.workflow.cs

```diff
@@ -0,0 +1,33 @@
+namespace TemporalioSamples.SleepForDays;
+
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+[Workflow]
+public class SleepForDaysWorkflow
```

Comment:
Class name should match pre-extension part of file name (should probably change file name in this case)

Reply:
Renamed file to `SleepForDaysWorkflow.workflow.cs`, looks a bit odd.

Reply:
Yeah, though that same seemingly-odd name is present on similar files/workflows in this repo